test/modules/arangojs.test.js

```diff
@@ -146,10 +145,9 @@ describe('Arangojs module ', async () => {
         assert.equal(retrievedVertex._key, vertexOne._key);
     });
 
-    it('trying to add same document again should resut in double insert', () => {
-        testDb.addVertex(vertexOne).then((response) => {
-            assert.equal(response, 'Double insert');
-        });
+    it('trying to add same document again should resut in double insert', async () => {
```

Comment:
seems we dont have 'double insert' as retrun value anymore...
please update the tittle